python/linked_list.py

```diff
@@ -1,19 +1,68 @@
-class LinkList:
-  # write your __init__ method here that should store a 'head' value which the first Node in the LinkedList and a 'length' value which is the total number of Nodes in the LinkedList
+class LinkList():
+  def __init__(self, head=None):
+    self.head = head
+    self.length = 1
+    print(self.head)
```

Comment:
It's probably best practice to remove all unnecessary print statements before turning it in